neard/res/genesis_for_test.json

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
The name doesn't explain what's different with this genesis. Should it explicitly call which test or protocol version?

Reply:
Done